charts/templates/marble-injector.yaml

```diff
@@ -99,31 +99,11 @@ webhooks:
       apiVersions: ["v1"]
       resources: ["pods"]
       scope: "Namespaced"
```

Comment:
Is the scope still "namespaced"?

Reply:
If I understand the kubernetes scope concept correctly a webhook with scope `"Namespaced"` is supposed to match resources that can be associated with namespaces (e.g. Pods). 
The two other options are `"Cluster"` for cluster-scoped resources (resources that do not belong to a namespace), and `""` which means there are no scope restrictions.

So `Namespaced` should still be correct

Reply:
Thanks for clearing that up!